assets/app/views/directives/_truncate.html

```diff
@@ -0,0 +1 @@
+<span ng-attr-title="{{content}}">{{content | limitTo : limit}}<span ng-if="content.length > (limit | toInt)">&hellip;</span></span>
```

Comment:
if you do the truncation in code to add word-preserving logic, you won't need the limitTo filter at all here